scripts/release/update_notebooks_bootstrapper.py

```
@@ -0,0 +1,65 @@
+    content = urllib.request.urlopen(url).read()
+    bp = old_dir / "elyra" / "kfp" / "bootstrapper.py"
+    bp.write_bytes(content)
+    print(f"  Updated {bp.relative_to(repo)}")
+
+    readme = old_dir / "elyra" / "kfp" / "README.md"
+    old_tag = old_name.removeprefix("elyra-")
+    readme.write_text(readme.read_text().replace(old_tag, tag))
+    print(f"  Updated {readme.relative_to(repo)}")
+
+    new_dir = prefetch / new_name
+    subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)